include/cuda/std/detail/libcxx/include/support/atomic/atomic_cuda.h

```diff
@@ -121,13 +121,43 @@ const volatile _Tp* __cxx_get_underlying_device_atomic(__cxx_atomic_base_heterog
 }
 
 template <typename _Tp>
-using __cxx_atomic_small_to_32 = typename conditional<is_signed<_Tp>::value, int32_t, uint32_t>::type;
+using __cxx_atomic_small_to_32 =
+    typename conditional<is_signed<_Tp>::value,
+        int32_t,
+        uint32_t
+    >::type;
+
+// Arithmetic conversions to/from proxy types
+template<class _Tp, _EnableIf<is_arithmetic<_Tp>::value, int> = 0>
+constexpr __host__ __device__ inline __cxx_atomic_small_to_32<_Tp> __cxx_small_to_32(_Tp __val) {
```

Comment:
Should this rather use `_LIBCUDACXX_INLINE_VISIBILITY`. Also I am not sure if we need the inline here,a s the function is already constexpr
```suggestion
_LIBCUDACXX_INLINE_VISIBILITY constexpr __cxx_atomic_small_to_32<_Tp> __cxx_small_to_32(_Tp __val) {
```

Reply:
Updated.